pom.xml

```diff
@@ -179,9 +179,18 @@
                 </executions>
                 <configuration>
                     <java>
+                        <includes>
+                            <!-- include all java files in "java" folders under "src" -->
+                            <include>src/**/java/**/*.java</include>
+                        </includes>
+
+                        <excludes>
+                            <exclude>src/main/java/org/rumbledb/parser/*</exclude>
```

Comment:
To answer your question on the 2nd point of the description. Yes includes and excludes statements work currently with spotless. However if we try to add any sort of ${basedir}/${project.basedir} etc. to the beginning of these relative paths, they stop working. 

These basedir variables work in other places of the pom.xml, just not in these include/exclude configs of spotless. I don't know the exact reason for this.